test/e2e/multikueue/e2e_test.go

```diff
@@ -165,6 +165,10 @@ var _ = ginkgo.Describe("MultiKueue", func() {
 	})
 
 	ginkgo.AfterEach(func() {
+		gomega.Expect(util.DeleteAllJobsetsInNamespace(ctx, k8sManagerClient, managerNs)).To(gomega.Succeed())
+		gomega.Expect(util.DeleteAllJobsetsInNamespace(ctx, k8sWorker1Client, worker1Ns)).To(gomega.Succeed())
+		gomega.Expect(util.DeleteAllJobsetsInNamespace(ctx, k8sWorker2Client, worker2Ns)).To(gomega.Succeed())
+
 		gomega.Expect(util.DeleteNamespace(ctx, k8sManagerClient, managerNs)).To(gomega.Succeed())
 		gomega.Expect(util.DeleteNamespace(ctx, k8sWorker1Client, worker1Ns)).To(gomega.Succeed())
 		gomega.Expect(util.DeleteNamespace(ctx, k8sWorker2Client, worker2Ns)).To(gomega.Succeed())
```

Comment:
Can we first check why `util.DeleteNamespace` is not sufficient?

Reply:
In `util.DeleteNamespace` we just have `DeleteAllJobsInNamespace` but don't have `DeleteAllJobsetsInNamespace`

Reply:
Moved this logic to `DeleteNamespace`.